dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/engine/command/impl/CreatePrivateUrlCommandTest.java

```diff
@@ -123,28 +123,30 @@ public void testAlreadyExists() {
     }
 
     @Test
-    public void testAttemptCreatePrivateUrlOnNonDraft() {
+    public void testCreatePrivateUrlSuccessfully_onReleased() {
         dataset = new Dataset();
```

Comment:
Suggestion: when refactoring it would be a good idea to divide a test into given/when/then sections and label them (here and in the other tests)